nix/pkgs/cryptobox/README.md

```diff
@@ -0,0 +1,7 @@
+# How to update
+
+```
+crate2nix source add git https://github.com/wireapp/cryptobox-c --rev $your-rev
+crate2nix generate
+```
+
```

Comment:
can we (eventually) integrate this into `./hack/bin/generate-local-nix-packages.sh` such that the overhead for non-nix-affine people is lower?

Reply:
It doesn't need to be regenerated, only if you consciously update the rev you want to use.

Also, the process isn't 100% automatic, you might need to adjust the nix expression, for example in case native build dependencies need to be added if it fails to build.

I'd rather keep this (more readable) documentation than just being hidden somewhere in a script doing another thing.

Reply:
> It doesn't need to be regenerated, only if you consciously update the rev you want to use.

oh yeah; these are external to wire-server anyways, yeah I think it makes sense like this :)

Reply:
That's not entirely true - a change in `Cargo.toml` `libzauth-c` would indeed update `Cargo.nix`.

I want to tackle this in a more general fashion in a followup, once the other two crates are migrated.